Cargo.toml

```diff
@@ -22,5 +22,6 @@ ink_lang = { git = "https://github.com/Phala-Network/ink.git", branch = "advtest
 ink_lang_codegen = { git = "https://github.com/Phala-Network/ink.git", branch = "advtest-3.4.0" }
```

Comment:
I suggest to reference the original ink crates but apply patches like what I've done in the offchain rollup repo. In that way, the patch will only apply in the test environment.